svm/src/account_loader.rs

```diff
@@ -74,7 +76,12 @@ impl Default for CheckedTransactionDetails {
         Self {
             nonce: None,
             lamports_per_signature: 0,
-            compute_budget_limits: Ok(ComputeBudgetLimits::default()),
+            compute_budget_and_limits: Ok(SVMTransactionComputeBudgetAndLimits {
+                budget: SVMTransactionExecutionBudget::default(),
+                loaded_accounts_bytes: NonZeroU32::new(32)
```

Comment:
ComputeBudgetLimits defaults this field to `MAX_LOADED_ACCOUNTS_DATA_SIZE_BYTES `, why changed to `32` here?